ui-tests/src/test/resources/default.properties

```diff
@@ -0,0 +1,3 @@
+login.url=https:/syndesis.192.168.xxx.xxx.nip.io
+login.username=x
+login.password=x
```

Comment:
Lets add prefix like, `syndesis.ui.url`, `syndesis.ui.username` to determine different properties for account credentials etc.